core/src/test/java/io/questdb/test/cutlass/pgwire/PGQuerySuspendTest.java

```diff
@@ -64,6 +65,12 @@ public static Collection<Object[]> testParams() {
 
     @Test
     public void testAllCases() throws Exception {
+        // limit cursor will not try to calculate size of the cursor proactively,
```

Comment:
'will not try' or 'will try' ?

Reply:
"now" sorry

Reply:
fixed in https://github.com/questdb/questdb/pull/5377/commits/f7861d6fe783a696d7b819712e2f9a0ebb6faa9d